packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -95,6 +95,7 @@ type Workspace {
   description: String
   createdAt: DateTime!
   updatedAt: DateTime!
+  logoUrl: String
```

Comment:
The logoUrl param in the WorkspaceCreateInput type doesn't make any sense. Whether we go with storing the base64 data string in the DB or store it in some kind of blob storage, the client is def not the one posting a logo URL to the server - they're gonna post the base64 string or post it as multiform data or whatever. The logo URL comes from the server (if the server creates one at all).

Reply:
+1, not sure where this came from, done